tests/reporting/test_market.py

```diff
@@ -19,7 +20,7 @@ def test_market_creation(contractsFixture):
     with raises(TransactionFailed, message="Shady share token should be failing since it is in a separate branch"):
         contractsFixture.createReasonableBinaryMarket(shadyBranch, shadyDenominationToken)
 
-    assert(market.getBranch() == branch.address)
+    assert(longToHexString(market.getBranch()) == branch.address)
```

Comment:
I didn't go too deep exploring exactly why this is occurring, but I suspect it has to do with the externs.sem entry for market being `getBranch:[]:int256`. I wanted to change externs.sem as little as possible though. This may change again when market is a Solidity file too.